src/mainboard/clevo/tgl-u/Kconfig

```diff
@@ -35,7 +35,6 @@ config BOARD_CLEVO_NV40MZ_BASE
 config BOARD_CLEVO_NS50MU_BASE
 	bool
 	select BOARD_CLEVO_TGLU_COMMON
-	select EC_SYSTEM76_EC_COLOR_KEYBOARD
```

Comment:
This was removed, because this symbol is not defined/used anywhere else. It was added in a8dacc0a1949c8082e489a5388aecf3ec2c701a3, but every commit after it never changed anything related to this symbol.

Reply:
Yes, this symbol was removed upstream. Can be removed.